.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
Why using source avoid caching the vendor folder? Also if you print the ENV variables you will get the token available, so anyone can get it.